docs/platform-qemu.md

```diff
@@ -1,15 +1,37 @@
 # LinuxKit with qemu/kvm
 
-TBD
+The `qemu` backend is the most versatile `run` backend for
+`linuxkit`. It can boot both `x86_64` and `arm64` images, runs on
+macOS and Linux (and possibly Windows), and can boot most types of
+output formats.
+
```

Comment:
Do we generally mention when a backend is the default on a platform (as qemu is for Linux)? Should we?

Reply:
we don't for hyperkit and hyper-v which are the only other defaults

Reply:
Also, do we/should we highlight here or elsewhere that this stuff is intended for dev test and not production usage?